src/Entity/Alias.php

```diff
@@ -26,12 +26,18 @@ class Alias implements SoftDeletableInterface
      */
     protected $destination;
 
+    /**
+     * @var bool
+     */
+    protected $random;
```

Comment:
Extract the property `$random` to a Trait, make the property private and add a parameter for `setRandom`